openbb_terminal/cryptocurrency/due_diligence/messari_view.py

```diff
@@ -48,33 +194,546 @@ def display_marketcap_dominance(
+            print_rich_table(
+                df_organizations,
+                headers=list(df_organizations.columns),
+                show_index=False,
+                title=f"{coin} Investors - Organizations",
+            )
+        else:
+            console.print("\nInvestors - Organizations not found\n")
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            "inv",
+            df_individuals,
+        )
+    else:
+        console.print("\nInvestors not found\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_MESSARI_KEY"])
+def display_team(
+    coin: str,
+    export: str = "",
+) -> None:
+    """Display coin team
+    [Source: https://messari.io/]
+
+    Parameters
+    ----------
+    coin : str
+        Crypto symbol to check coin team
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+    (df_individuals, df_organizations) = get_team(coin)
+    if not df_individuals.empty or not df_organizations.empty:
+        if not df_individuals.empty:
+            print_rich_table(
+                df_individuals,
+                headers=list(df_individuals.columns),
+                show_index=False,
+                title=f"{coin} Team - Individuals",
+            )
+        else:
+            console.print("\nIndividual team members not found\n")
+        if not df_organizations.empty:
+            print_rich_table(
+                df_organizations,
+                headers=list(df_organizations.columns),
+                show_index=False,
+                title=f"{coin} Team - Organizations",
+            )
+        else:
+            console.print("\nTeam organizations not found\n")
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            "team",
+            df_individuals,
+        )
+    else:
+        console.print("\nTeam not found\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_MESSARI_KEY"])
+def display_governance(
+    coin: str,
+    export: str = "",
+) -> None:
+    """Display coin governance
+    [Source: https://messari.io/]
+
+    Parameters
+    ----------
+    coin : str
+        Crypto symbol to check coin governance
+    export : str
+        Export dataframe data to csv,json,xlsx file
+    """
+    (summary, df) = get_governance(coin)
+    if summary:
+        console.print(summary, "\n")
+        if not df.empty:
+            print_rich_table(
+                df,
+                headers=list(df.columns),
+                show_index=False,
+                title=f"{coin} Governance details",
+            )
+        export_data(
+            export,
+            os.path.dirname(os.path.abspath(__file__)),
+            "gov",
+            df,
+        )
+    else:
+        console.print(f"\n{coin} governance details not found\n")
+
+
+@log_start_end(log=logger)
+@check_api_key(["API_MESSARI_KEY"])
+def display_fundraising(
```

Comment:
Any idea why the investor % and org % doesn't match between the chart and text? With ETH it's matching tho so might be due to Messari itself?
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/40023817/165496281-b8260992-c31a-4cb9-9df3-16b4626cfd25.png">